spec/e2e/unpersisted_association_spec.rb

```diff
@@ -221,7 +221,11 @@ def save_and_expect_rel!
       it 'does not raise error, creates rel on save' do
         expect_any_instance_of(Neo4j::Core::Query).not_to receive(:delete)
         expect { chris.lesson_ids = [math.id] }.not_to raise_error
-        expect { chris.save }.to change { chris.lessons.count }
+        expect { chris.save }.not_to change { chris.lessons.count }
+      end
+
+      it 'changes count' do
```

Comment:
You might save someone a moment of consideration in the future if you changed this to just `it 'changes count, even before persisting the node'` or something to be super clear about this example. (Ignore this if an earlier context makes it clear!)